frontend/src/libs/packages/services/notification.ts

```diff
@@ -0,0 +1,30 @@
+import { toast } from 'react-toastify';
```

Comment:
path should be `libs/packages/notification/notification.package.ts`, not `libs/packages/services/notification.ts`

and then we need to create an instance of this class in `libs/packages/notification/notification.ts` and re-export it as well as NotificationType